libraries/tools/kotlin-gradle-plugin/src/common/kotlin/org/jetbrains/kotlin/gradle/targets/native/cocoapods/KotlinCocoapodsPlugin.kt

```diff
@@ -371,6 +371,7 @@ open class KotlinCocoapodsPlugin : Plugin<Project> {
             it.license.set(cocoapodsExtension.license)
             it.authors.set(cocoapodsExtension.authors)
             it.summary.set(cocoapodsExtension.summary)
+            it.source.set(cocoapodsExtension.source)
```

Comment:
@jeffdgr8 

Your PR has conflict because this file has been changed and the podspec generation logic has been moved to `fun PodspecTask.configure`. please rebase your branch and apply this code based on latest changes. Thanks

Reply:
Also I found a [similar thread](https://youtrack.jetbrains.com/issue/KT-52784/KMM-Cocoapods-Podspec-Task-ignores-kotlin.cocoapods.source-parameter#focus=Comments-27-6192160.0-0) on YouTrack. It seems not copying the source in the project podspec is a expected behavior. because it should only use for local XCode and Kotlin project integration. there is another gradle task named "podPublishReleaseXCFramework" that genrates another podspec file with the specified source field in the directory "shared/build/cocoapods/publish/release". this file must be used for publication purpose.

if I figure it correctly, then this PR can be closed and the change is not necessary.